### PR TITLE
Configuration of DB authentication instead of In memory authentication

### PR DESCRIPTION
**Pull request by julia-shvabovich**

### CODE REVIEW

File: src/main/java/cinema/inject/Initializer.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        admin.setRoles(Set.of(roleService.getRoleByName("USER"),
+                roleService.getRoleByName("ADMIN")));
+        userService.add(admin);
+
+        User user = new User();
+        user.setEmail("secretmail@gmail.com");
+        user.setPassword("54321");
+        user.setRoles(Set.of(roleService.getRoleByName("USER")));
+        userService.add(user);
+    }
```
Comment (fyarkiy) [inline src/main/java/cinema/inject/Initializer.java:40]: Why would you set "dummy" user through init? What is the purpose?

**Review by fyarkiy — APPROVED** [review]

**Comment by fyarkiy** [comment]
Good job!

File: src/main/java/cinema/inject/Initializer.java (hunk 1 above)
Comment (julia-shvabovich) [reply src/main/java/cinema/inject/Initializer.java:40]: i don't understand what you mean...

File: src/main/java/cinema/config/AppConfig.java [hunk 2]
```diff
@@ -9,14 +9,17 @@
...
+import org.springframework.security.crypto.password.PasswordEncoder;
 
 @Configuration
 @PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "cinema.service.impl",
         "cinema.dao.impl",
         "cinema.security",
-        "cinema.model.dto"
+        "cinema.model.dto",
```
Comment (Sofasmile) [inline src/main/java/cinema/config/AppConfig.java:21]: should we add this package here?

**Review by Sofasmile — APPROVED** [review]

**Pull request merged**